include/rocksdb/perf_context.h

```diff
@@ -26,6 +28,10 @@ struct PerfContextByLevel {
   // exist.
   uint64_t bloom_filter_full_true_positive = 0;
 
+  uint64_t user_key_return_count;  // total number of user key returned
+
+  uint64_t get_processing_time;  // total nanos spent on Get query
```

Comment:
We used to do ..._time, but later realized that it was not good. Now the convention is `..._nanos`. This can be something like `get_from_table_nanos`.